the-event-loop/interval.js

```diff
@@ -0,0 +1,11 @@
+let count = 3;
+const intervalId = setInterval(() => {
+  if (count === 0) {
```

Comment:
(Optional) Though it doesn't really matter here, but there is a potential problem if for some reason this call doesn't happen when count is 0, because the 'else' clause will just count down to -1, -2, etc., forever. For safety, it's good to check `<= 0` instead of `=== 0`.